crates/nu-command/tests/commands/do_.rs

```diff
@@ -80,7 +80,7 @@ fn capture_error_with_too_much_stderr_not_hang_nushell() {
         let actual = nu!(
             cwd: dirs.test(), pipeline(
             r#"
-            do -c {sh -c "cat a_large_file.txt 1>&2"} | complete | get stderr
+            sh -c "cat a_large_file.txt 1>&2" | complete | get stderr
```

Comment:
Without `do` they should probably located in a different place.

Reply:
Good catch!